packages/client/test/end-to-end/StreamrClient.test.ts

```diff
@@ -20,11 +18,11 @@ describe('StreamrClient', () => {
     })
 
     it('getPeerDescriptor', async () => {
-        await client.subscribe('foobar')
         const descriptor = await client.getPeerDescriptor()
         expect(descriptor).toMatchObject({
-            id: expect.stringMatching(UUID_REGEX),
+            id: expect.toBeString(),
```

Comment:
This checks the implementation detail of `PeerID`. Maybe any string is a valid `NetworkPeerDescriptor#id` value, as there is no specific format that the data is? (It it is maybe just semi-random bytes)

Reply:
It used to check the implementation detail (i.e. that it is an UUID). Now it just checks that it is any string, which is less of an implementation detail?